server/services/withings/api/withings.controller.js

```diff
@@ -0,0 +1,59 @@
+const asyncMiddleware = require('../../../api/middlewares/asyncMiddleware');
+
+module.exports = function WithingsController(withingsHandler) {
+  /**
+   * @description Init gladys devices with withings devices.
+   * @api {post} /api/v1/service/withings/init Init gladys devices with withings devices.
+   * @apiName init
+   * @apiGroup Withings
+   */
+  async function init(req, res) {
+    if (req.user && req.user.id) {
```

Comment:
Maybe we didn't understand each other, but this code will never be false, because if we are in an authenticated function, req.user.id always exist.